.pre-commit-config.yaml

```diff
@@ -25,7 +25,12 @@ repos:
     hooks:
     -   id: flake8
         language_version: python3
-        additional_dependencies: [flake8-typing-imports==1.3.0]
+        additional_dependencies: [
+          flake8-typing-imports==1.3.0,
+          # To handle PEP 484 type comments with following text.
+          # (https://github.com/PyCQA/pyflakes/pull/455)
+          'git+https://github.com/PyCQA/pyflakes.git@1911c20#egg=pyflakes',
```

Comment:
Maybe we should wait until it's officially released? For example in my development venv I just do `pip install flake8` which means my editor will still complain. Also maybe things will change before the release.

Reply:
Yeah.. but from what I can tell it might still take months, and we should not let that hold us back.
Let's ignore the flake8 error then instead?  (this is what mypy itself does)

Reply:
> Let's ignore the flake8 error then instead?

:+1: 

Next flake8 release will be nice for all the typing fixes :)